online-banking-cz/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 23/8/2022.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+ext.mavenArtifactId = "online-banking-cz"
+ext.mavenArtifactName = "Adyen checkout Online Banking Czech"
```

Comment:
Maybe we can ask product about `Online Banking Czech` vs `Online Banking Czech Republic` and change it everywhere if needed.

Reply:
it's "Czech republic" inside the ticket actually, so it's just small change will do it now